src/app/modules/timesheet/services/timesheet.service.ts

```diff
@@ -120,14 +120,44 @@ export class TimesheetService {
   }
 
   public openLastTimesheetInLocal(): boolean {
-    const timesheetsOfLocalStorage = this.getTimesheetsLocal();
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
     if (timesheetsOfLocalStorage.length > 0) {
       this.setTimesheet(this._localSaveService.getLocalItem(timesheetsOfLocalStorage[timesheetsOfLocalStorage.length - 1]));
       return true;
     }
     return false;
   }
 
+  public getPresentTimesheet(transferedTimesheet: Timesheet) {
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
+    const localStorageSize = this.getLocalstorageTimesheetsList().length;
+    for (let i = localStorageSize; i >= 0; i--) {
+      const transfertTimesheet: Timesheet = Object.assign({},
```

Comment:
`transfertTimesheet` is not clear, what you want to say ? `alreadyTranferedTimesheet`, `previousTranferedTimesheet`, `tranferedTimesheet` or something like that ?

Reply:
Yes something like that, not clear it's true

Reply:
It's nice yes :)